### PR TITLE
Marcio/1.1.1.1 hostnames tls

### PR DESCRIPTION
**Pull request by marciocloudflare**
Added information that DNS over TLS resolver can also be reached by hostname

### CODE REVIEW

**Review by dcpena — APPROVED** [review]

File: products/1.1.1.1/src/content/encrypted-dns/dns-over-tls.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ By default, DNS is sent over a plaintext connection. DNS over TLS (DoT) is one w
 
 ## How it works
 
-Cloudflare supports DNS over TLS on `1.1.1.1` and `1.0.0.1` on port 853. The certificate presented is for `cloudflare-dns.com`. A stub resolver (the DNS client on a device that talks to the DNS resolver) connects to the resolver over a TLS connection:
+Cloudflare supports DNS over TLS on `1.1.1.1` and `1.0.0.1` on port 853. Cloudflare's 1.1.1.1 DNS resolver can also be reached by hostname on `1dot1dot1dot1.cloudflare-dns.com` and `one.one.one.one`. The certificate presented is for `cloudflare-dns.com`. A stub resolver (the DNS client on a device that talks to the DNS resolver) connects to the resolver over a TLS connection:
```
Comment (elithrar) [inline products/1.1.1.1/src/content/encrypted-dns/dns-over-tls.md:12]: > Cloudflare supports DNS over TLS on ...

"Cloudflare supports DNS over TLS (DoT) on ..."

Comment (elithrar) [inline products/1.1.1.1/src/content/encrypted-dns/dns-over-tls.md:12]: > Cloudflare's 1.1.1.1 DNS resolver can ...

"The DoT resolver can also be reached by hostname, if your client does not support IP addresses, on ..."

Comment (elithrar) [inline products/1.1.1.1/src/content/encrypted-dns/dns-over-tls.md:12]: > The certificate presented is for `cloudflare-dns.com`

Not true - the certificate for all of the names.

```
X509v3 Subject Alternative Name:
                DNS:cloudflare-dns.com, DNS:*.cloudflare-dns.com, DNS:one.one.one.one, IP Address:1.1.1.1, IP Address:1.0.0.1, IP Address:162.159.36.1, IP Address:162.159.46.1, IP Address:2606:4700:4700:0:0:0:0:1111, IP Address:2606:4700:4700:0:0:0:0:1001, IP Address:2606:4700:4700:0:0:0:0:64, IP Address:2606:4700:4700:0:0:0:0:6400
```

**Review by elithrar — COMMENTED** [review]
Some minor comments to make it clear "why" you might want to use the hostname.

**Comment by elithrar** [comment]
@marciocloudflare Can you re-open / incorporate these comments? It's only been ~24 hours since I was added for review.

**Pull request merged**